scripts/prstate

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/python3
+# SPDX-License-Identifier: Apache-2.0
+
+from github import Github
+import os
+
+github = Github(os.environ['GITHUB_TOKEN'])
+
+for issue in github.search_issues(f"org:enarx state:open is:public is:pr"):
+    pr = issue.as_pull_request()
+
+    reviewers = {r.user.login for r in pr.get_reviews() if r.state == 'CHANGES_REQUESTED'}
+    requested = {r.login for r in pr.get_review_requests()[0]}
```

Comment:
Wouldn't fetching index 0 here only look at the first review request?

Reply:
No. Read the docs for the function. :)

Reply:
Aha! My brain parsed that wrong when I looked at it the first time. Carry on.